src/main/java/com/google/firebase/remoteconfig/internal/TemplateResponse.java

```diff
@@ -39,6 +39,9 @@
   @Key("version")
   private VersionResponse version;
 
+  @Key("etag")
```

Comment:
Add a comment here indicating that this is never set by the HTTP response payload.